app/views/subtopics/_subtopic.html.erb

```diff
@@ -11,17 +11,17 @@
       <% end %>
     </div>
 
-    <div 
-      class="govuk-grid-column-one-third govuk-!-margin-top-8 govuk-!-margin-bottom-3"
-      data-module="ga4-link-tracker"
-      data-ga4-link='{ "event_name": "navigation", "type": "subscribe", "index": { "index_link": 1 }, "index_total": 1, "section": "Top" }'
-      data-ga4-track-links-only
-    >
-      <%= render "govuk_publishing_components/components/subscription_links", {
-        email_signup_link: "/email-signup?link=/topic/#{params[:topic_slug]}/#{params[:subtopic_slug]}",
-        email_signup_link_text: t("shared.get_emails")
-      } %>
-
+    <div class="govuk-grid-column-one-third govuk-!-margin-top-8 govuk-!-margin-bottom-3">
+      <span
+        data-module="ga4-link-tracker"
+        data-ga4-link='{ "event_name": "navigation", "type": "subscribe", "index": { "index_link": 1 }, "index_total": 1, "section": "Top" }'
+        data-ga4-track-links-only
```

Comment:
I think you could also pass the attributes directly to the component? https://components.publishing.service.gov.uk/component-guide/subscription_links#with_tracking

Reply:
@andysellick Thanks. I spoke to James and I think there's a reason the tracking is added like this. We couldn't remember fully, but think it was something to do with multiple types of links in the component, and the need to track them separately/with different values? 

Here are the original PRs https://github.com/alphagov/finder-frontend/pull/3081 and https://github.com/alphagov/finder-frontend/pull/3081

Would you recommend I still try and pass it through the component? Thanks

Reply:
No if you have a reason for doing it this way, that's fine.